architecture-building-systems/CityEnergyAnalyst
# 904 config file editor

Implements a first stab at a config file editor for the user configuration file.

How to test:

- (you might need to `conda env update` if you get errors about packages missing)
- also do `pip install -e .[dev]`
- run `cea config-editor`

You should see a GUI that let's you edit the options of each section in the config file. Clicking the `Save` button saves the current section to disk.
